### ISSUE TITLE
bodhi-backend02 is unable to lookup updates by alias

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1473]
Message
-------
[2017-04-25 02:43:33][    fedmsg   ERROR]
```python
{'body': {u'username': u'bodhi', u'certificate': u'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVUakNDQTdlZ0F3SUJBZ0lDQWdnd0RRWUpL\nb1pJaHZjTkFRRUZCUUF3Z2FBeEN6QUpCZ05WQkFZVEFsVlQKTVFzd0NRWURWUVFJRXdKT1F6RVFN\nQTRHQTFVRUJ4TUhVbUZzWldsbmFERVhNQlVHQTFVRUNoTU9SbVZrYjNKaApJRkJ5YjJwbFkzUXhE\nekFOQmdOVkJBc1RCbVpsWkcxelp6RVBNQTBHQTFVRUF4TUdabVZrYlhObk1ROHdEUVlEClZRUXBF\nd1ptWldSdGMyY3hKakFrQmdrcWhraUc5dzBCQ1FFV0YyRmtiV2x1UUdabFpHOXlZWEJ5YjJwbFkz\nUXUKYjNKbk1CNFhEVEUxTURjeU1URTVNVE15TTFvWERUSTFNRGN4T0RFNU1UTXlNMW93Z2R3eEN6\nQUpCZ05WQkFZVApBbFZUTVFzd0NRWURWUVFJRXdKT1F6RVFNQTRHQTFVRUJ4TUhVbUZzWldsbmFE\nRVhNQlVHQTFVRUNoTU9SbVZrCmIzSmhJRkJ5YjJwbFkzUXhEekFOQmdOVkJBc1RCbVpsWkcxelp6\nRXRNQ3NHQTFVRUF4TWtZbTlrYUdrdFltOWsKYUdrd015NXdhSGd5TG1abFpHOXlZWEJ5YjJwbFkz\nUXViM0puTVMwd0t3WURWUVFwRXlSaWIyUm9hUzFpYjJSbwphVEF6TG5Cb2VESXVabVZrYjNKaGNI\nSnZhbVZqZEM1dmNtY3hKakFrQmdrcWhraUc5dzBCQ1FFV0YyRmtiV2x1ClFHWmxaRzl5WVhCeWIy\ncGxZM1F1YjNKbk1JR2ZNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0R05BRENCaVFLQmdRQ28KcjF1MmE3\nU20vcGhMMG5
 lZG9FZXNyUUZDRGZVYmMwdGt1bUowQlBiRHcxdzdtblp0MERVSjhrNkVXVFNwL25E\nNwpoK0NNMmRucG5lRm5XVllycDlXSGpSUVZEbzBCemE3MnJQRjEwcjhPb3phN3k3VTBFUlhmMkNy\nZnBxZTJzREpNCkFyWThZZnZVWnluemIwVUVzOFlmNkgrQ2h5S1JldGNXaWF1TTBZQU5zUUlEQVFB\nQm80SUJWekNDQVZNd0NRWUQKVlIwVEJBSXdBREF0QmdsZ2hrZ0JodmhDQVEwRUlCWWVSV0Z6ZVMx\nU1UwRWdSMlZ1WlhKaGRHVmtJRU5sY25ScApabWxqWVhSbE1CMEdBMVVkRGdRV0JCUmQyMWNBdnZJ\ncFlrQWlIR05nUmRpdHdMODZLVENCMVFZRFZSMGpCSUhOCk1JSEtnQlJyUUZyNUVnaUpXZWRaNVFY\nMUFoMEtUbjhVQUtHQnBxU0JvekNCb0RFTE1Ba0dBMVVFQmhNQ1ZWTXgKQ3pBSkJnTlZCQWdUQWs1\nRE1SQXdEZ1lEVlFRSEV3ZFNZV3hsYVdkb01SY3dGUVlEVlFRS0V3NUdaV1J2Y21FZwpVSEp2YW1W\namRERVBNQTBHQTFVRUN4TUdabVZrYlhObk1ROHdEUVlEVlFRREV3Wm1aV1J0YzJjeER6QU5CZ05W\nCkJDa1RCbVpsWkcxelp6RW1NQ1FHQ1NxR1NJYjNEUUVKQVJZWFlXUnRhVzVBWm1Wa2IzSmhjSEp2\nYW1WamRDNXYKY21lQ0NRRGpVQjVIVHhjZVJUQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFM\nQmdOVkhROEVCQU1DQjRBdwpEUVlKS29aSWh2Y05BUUVGQlFBRGdZRUFieTE3QVExbGhzckxuamZk\nWEhaWG9Kd2pWcVdTNHRudWZvVzFLK3pLCnZiTkdMNlBmQVZHa3Q5RzlENENpQzV6amlPNzdi
 Y2lX\nTkYzeWdUUjZqcmdoUnJOZmZnQ1MzSWFpT3JNemVnaHYKeWRqcXJqdDBDY3VoRXM3WFRndVlHcWoz\ndThxS3J6ZGVtNGxETm9wNENoZ0ZtS1ZUdk1pc3R4UVdIMEhCdDZkcApZYXM9Ci0tLS0tRU5EIENF\nUlRJRklDQVRFLS0tLS0K\n', u'i': 2, u'timestamp': 1493088212, u'msg_id': u'2017-f9a6f510-50b5-4612-b5be-a96fa11c6135', u'crypto': u'x509', u'topic': u'org.fedoraproject.prod.bodhi.update.request.testing', u'signature': u'X9KIpc/uXZPiCH6HTcC+jk1FlN6iBVfhv7CS/PKxcBAQ4Kvrehjiqs0j5BHSfAiD2gfj0eLPeaoB\nKqvXJLWr3R87+xO/FHuJ+Sk7b4GeS9ANcRAY4yH6Gmc0c42Gti2U4FHwOS7JPQEoEZhmNB9kx6cE\nh4XAtTjlMAAEwq/T8a4=\n', u'msg': {u'update': {u'date_testing': None, u'old_updateid': None, u'pushed': False, u'require_testcases': True, u'date_stable': None, u'critpath': False, u'date_approved': None, u'stable_karma': 3, u'date_pushed': None, u'requirements': u'', u'severity': u'unspecified', u'autokarma': True, u'title': u'xrdp-0.9.2-10.fc25', u'suggest': u'unspecified', u'require_bugs': True, u'comments': [{u'bug_feedback': [], u'user_id': 91, u'times
 tamp': u'2017-04-25 02:43:31', u'text': u'This update has been submitted for testing by bojan. ', u'karma_critpath': 0, u'update_id': 86094, u'karma': 0, u'anonymous': False, u'testcase_feedback': [], u'id': 598316, u'user': {u'openid': None, u'name': u'bodhi', u'show_popups': True, u'email': None, u'avatar': None, u'groups': [], u'id': 91}}], u'updateid': u'FEDORA-2017-dfced1cc2a', u'test_cases': [], u'close_bugs': True, u'meets_testing_requirements': False, u'date_locked': None, u'date_submitted': u'2017-04-25 02:43:31', u'unstable_karma': -3, u'submitter': u'bojan', u'user': {u'openid': None, u'name': u'bojan', u'show_popups': True, u'email': u'bojan@rexursive.com', u'avatar': None, u'groups': [{u'name': u'packager'}], u'id': 198}, u'locked': False, u'builds': [{u'epoch': 0, u'nvr': u'xrdp-0.9.2-10.fc25', u'signed': False}], u'date_modified': None, u'url': u'https://bodhi.fedoraproject.org/updates/FEDORA-2017-dfced1cc2a', u'type': u'enhancement', u'notes': u'SELinux related impro
 vements:\n\n- move scripts to /usr/libexec/xrdp, so that they get labelled as bin_t\n- add SELinux policy sub-package', u'request': u'testing', u'bugs': [], u'alias': u'FEDORA-2017-dfced1cc2a', u'status': u'pending', u'karma': 0, u'release': {u'dist_tag': u'f25', u'name': u'F25', u'testing_tag': u'f25-updates-testing', u'pending_stable_tag': u'f25-updates-pending', u'pending_signing_tag': u'f25-signing-pending', u'long_name': u'Fedora 25', u'state': u'current', u'version': u'25', u'override_tag': u'f25-override', u'branch': u'f25', u'id_prefix': u'FEDORA', u'pending_testing_tag': u'f25-updates-testing-pending', u'stable_tag': u'f25-updates', u'candidate_tag': u'f25-updates-candidate'}}, u'agent': u'bojan'}}, 'topic': u'org.fedoraproject.prod.bodhi.update.request.testing'}
```

Process Details
---------------
- host:     bodhi-backend02.phx2.fedoraproject.org
- PID:      20075
- name:     fedmsg-hub
- command:  /usr/bin/python /usr/bin/fedmsg-hub
- msg_id:   

Callstack that lead to the logging statement
--------------------------------------------
```python
  File "/usr/lib64/python2.7/threading.py", line 784 in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib64/python2.7/threading.py", line 811 in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764 in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/twisted/python/threadpool.py", line 167 in _worker
    result = context.call(ctx, function, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/twisted/python/context.py", line 118 in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/usr/lib64/python2.7/site-packages/twisted/python/context.py", line 81 in callWithContext
    return func(*args,**kw)
  File "/usr/lib/python2.7/site-packages/moksha/hub/api/consumer.py", line 193 in _work
    self.log.exception(message)
```

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/moksha/hub/api/consumer.py", line 191, in _work
    self.consume(message)
  File "/usr/lib/python2.7/site-packages/bodhi/server/consumers/updates.py", line 96, in consume
    raise BodhiException("Couldn't find alias %r in DB" % alias)
BodhiException: Couldn't find alias u'FEDORA-2017-dfced1cc2a' in DB
```

**Comment by bowlofeggs** [comment]
At this point, it seems likely that this is the same underlying issue as #1470, and I believe the fix I've got for it will fix this too.

**Comment by bowlofeggs** [comment]
This fix for this has been cherry-picked to the 2.6 branch as 1566e99edcaa4ee2246072f1a620f32340560c58.
